CHANGELOG.md

```diff
@@ -2,54 +2,74 @@
 
 # Versioning
 
-NB: This gem is versioned based on the `MAJOR`.`MINOR` version of rubocop. The first release of the
+ezCater is using [Semantic Versioning](https://semver.org/) moving forward with v1.0.0. All version bumps moving forward should increment using `MAJOR.MINOR.PATCH` based on changes.
```

Comment:
🎱 thoughts on saying something like 'moving onto its own semantic versioning scheme' or something similar? some wording that makes it more pronounced that 'we were doing rubocop's versions, now they are ours'